Patterns/English/English-DateTime.yaml

```diff
@@ -513,7 +513,7 @@ BeforeRegex: !nestedRegex
   references: [ InclusiveModPrepositions ]
 # "starting from" is SinceRegex
 AfterRegex: !nestedRegex
-  def: (\b{InclusiveModPrepositions}?((after|(starting|beginning)(\s+on)?(?!\sfrom)|(?<!no\s+)later than)|(year greater than))(?!\s+or equal to){InclusiveModPrepositions}?\b\s*)|(?<!\w|<)((?<include>>=)|>)
+  def: ((\b{InclusiveModPrepositions}?((after|(starting|beginning)(\s+on)?(?!\sfrom)|(?<!no\s+)later than)|(year greater than))(?!\s+or equal to){InclusiveModPrepositions}?\b\s*?)|(?<!\w|<)((?<include>>=)|>))(\s+the)?

```

Comment:
`*` means zero or more `\s`, why do we even need a `?` here?

Reply:
*? means Lazy quantifier. Appending the ? character to a quantifier makes it lazy; it causes the regular expression engine to match as few occurrences as possible. 
So that we can correctly match the "\s+the" in the behind

Reply:
That's reasonable.